components/infobox/extensions/commons/placement_stats.lua

```diff
@@ -0,0 +1,133 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:InfoboxPlacementStats
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local placementStats = {}
+local Class = require('Module:Class')
+local Template = require('Module:Template')
+local Table = require('Module:Table')
+local Medal = require('Module:Medal')
+local Settings = require('Module:InfoboxPlacementStats/Settings')
+
+local _PLACEMENTS = { '1', '2', '3', '>' }
```

Comment:
This var could use a comment explaining why it exists, it is very hard to figure out.